docs/plugins/testing_workshop_docs/article-1-pythons-assert-keyword.md

```diff
@@ -0,0 +1,93 @@
+# Article 1: Python's assert keyword
+
+This article defines the assert keyword in Python and shows how it can be used to write test cases for a simple function.  
+
+## Article information  
+
+* Article 1: This article (Python's assert keyword) 
+* Article 2: [Pytest testing framework](./article-2-pytest-testing-frameworks)  
+* Article 3: [Readers and fixtures](./article-3-readers-and-fixtures)  
+* Article 4: [Test coverage](./article-4-test-coverage)  
+* Resource links: [Testing resources](./testing-resources)  
+  
+### This article covers:  
+[Assert keyword](#assert-keyword)  
+[Test for Pass](#test-for-pass)  
```

Comment:
This link is not currently working, I think it needs to be updated to match the header so:

```suggestion
[Test for Pass](#test-for-the-pass-case)  
```

Reply:
I think this might be my fault 😓 

I suggested a restructure of this page from:

## Test for pass
## Test for fail

to:

## Example tests using assert
### Test for the pass case
### Test for the fail case

I think that a mix of keeping the original structure and changing to the suggested structure might have happened